src/utils/decorators/input-validation/input-validation.js

```diff
@@ -633,7 +634,10 @@ const InputValidation = (ComposedComponent) => {
 
       const errorMessage = (this.state.messageLocked || this.state.messageShown) && (
         <Portal key='1' onReposition={ this.positionMessage }>
-          <div className='common-input__message-wrapper'>
+          <div
+            className='common-input__message-wrapper'
+            style={ { pointerEvents: 'none' } }
```

Comment:
It'd be better to add `pointer-event` to the `input.scss` file where the `.common-input__message-wrapper` is class is defined.